workers/unity/Packages/io.improbable.gdk.deploymentlauncher/Tests/Editmode/DeploymentConfig/DeploymentConfigTests.cs

```diff
@@ -0,0 +1,46 @@
+using System.Linq;
+using Improbable.Gdk.DeploymentLauncher.EditmodeTests.Utils;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace Improbable.Gdk.DeploymentLauncher.EditmodeTests
+{
+    [TestFixture]
+    public class DeploymentConfigTests
+    {
+        [Test]
+        public void Valid_config_gives_no_errors()
+        {
+            Assert.IsFalse(DeploymentConfigTestUtils.GetValidDeploymentConfig().GetErrors().Any());
+        }
+
+        [TestCase("gdk_test_assembly")]
+        public void Valid_assembly_name_test(string assemblyName)
```

Comment:
would change naming of all the tests slightly. 
e.g. `DeploymentConfig_throws_error_when_invalid_assembly_name`
or `DeploymentConfig_has_valid_assembly_name`

think these are the naming conventions we have used in the past and makes it easy to see what we are testing and the expected outcome

Reply:
tests have since been renamed 👍